apps/frontend/src/containers/Project/ConnectRepository.tsx

```diff
@@ -385,6 +443,14 @@ export const ConnectRepository = (props: ConnectRepositoryProps) => {
           >
             Continue with GitHub
           </GitHubButton>
+          {ghLightGhInstallation && (
+            <GitHubBaseButton
+              size="large"
+              onPress={() => setAndStoreProvider(GitProvider.GitHubLight)}
```

Comment:
One is "large" and not the other one. Is it normal?

Reply:
Yeah